src/Tes.Runner/Docker/NetworkUtility.cs

```diff
@@ -101,6 +102,14 @@ private async Task<(string Output, string Error)> RunIptablesCommandAsync(string
                 case 0:
                     return (output, error);
                 case 4:
+                    if (error?.Contains("Another app is currently holding the xtables lock", StringComparison.OrdinalIgnoreCase) == true)
+                    {
+                        // "Another app is currently holding the xtables lock. Perhaps you want to use the -w option?"
+                        var invalidOperationException = new InvalidOperationException($"xtables is locked by another app. Error: {error}");
```

Comment:
I'm interested in knowing if the caller to the methods in this class is retrying these calls, and if so, which exceptions result in retries.

The answer isn't needed to merge this code, though

Reply:
No they are not being retried anywhere